.gitignore

```diff
@@ -9,5 +9,4 @@ package-lock.json
 *.rdb
 .env*
 localStorage
-schema.graphql
```

Comment:
we need this file Danial so that we can import the schema to postman and generate tests

Reply:
Reverted.